### PR TITLE
 Upgrade Python syntax and unit test asserts

### PR DESCRIPTION
**Pull request by hugovk**
* Upgraded Python syntax with [`pyupgrade --py36-plus`](https://github.com/asottile/pyupgrade).
* Upgraded unit tests to use more useful asserts.

### CODE REVIEW

File: lbrynet/wallet/claim_proofs.py [hunk 1]
```diff
@@ -52,9 +52,9 @@ def verify_proof(proof, rootHash, name):
         if i == 0 and 'txhash' in proof and 'nOut' in proof and 'last takeover height' in proof:
             if len(proof['txhash']) != 64:
                 raise InvalidProofError("txhash was invalid: {}".format(proof['txhash']))
-            if not isinstance(proof['nOut'], six.integer_types):
+            if not isinstance(proof['nOut'], (int,)):
```
Comment (eukreign) [inline lbrynet/wallet/claim_proofs.py]:
```suggestion
            if not isinstance(proof['nOut'], int):
```

File: lbrynet/wallet/claim_proofs.py [hunk 2]
```diff
@@ -52,9 +52,9 @@ def verify_proof(proof, rootHash, name):
...
                 raise InvalidProofError("nOut was invalid: {}".format(proof['nOut']))
-            if not isinstance(proof['last takeover height'], six.integer_types):
+            if not isinstance(proof['last takeover height'], (int,)):
```
Comment (eukreign) [inline lbrynet/wallet/claim_proofs.py]:
```suggestion
            if not isinstance(proof['last takeover height'], int):
```

**Comment by eukreign** [comment]
@hugovk thanks for your two PRs, we've merged one and will merge this one after we do a currently pending 0.30 release today (to give us some time to make sure this massive change doesn't have any negative impact).

Can we send you some LBC as a thank you for your effort?

**Comment by tzarebczan** [comment]
@hugovk Thanks for the contribution! Believe I replied in another one of your Prs- but send us an email at hello@lbry.io (mention this PR) and we'll get a form over to you to fill out for the t-shirt :)

**Pull request merged**